### PR TITLE
Feature/pxweb2 158 radio

### PR DESCRIPTION
**Pull request by KentMossback**
Added Radio component and used it in Select.tsx

### CODE REVIEW

File: libs/pxweb2-ui/src/lib/components/Select/Select.tsx [hunk 1]
```diff
@@ -220,7 +225,17 @@ function VariableBoxSelect({
...
+                <Radio 
+                  value={clickedItem?.value ?? ''}
+                  name='option'
+                  options={options}                  
+                  selectedOption={clickedItem?.value}
+                  onChange={handleRadioChange}
+                >                
+                </Radio>
+
+         {/* <div className={cl(classes.modalRadioList)}>
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Select/Select.tsx:238]: Should this commented out code be removed?

File: libs/pxweb2-ui/src/lib/components/Select/Select.tsx [hunk 2]
```diff
@@ -220,7 +225,17 @@ function VariableBoxSelect({
           isOpen={isModalOpen}
           onClose={handleCloseModal}
         >
-          <div className={cl(classes.modalRadioList)}>
+
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Select/Select.tsx:228]: Seems like the indentation is off here, can you format the document?

**Comment by SjurSutterudSagen** [comment]
When I look in Storybook it looks good, but when I go to the branch preview and look at it in the variablebox the checked mark is not centered in the middle of the circle.

**Review by MikaelNordberg — CHANGES_REQUESTED** [review]
The functionality seems to be working but you should fix the css stuff that i mentioned

File: libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.divider {
+  padding: 10px var(--spacing-2, 8px);
+  align-items: flex-start;
+  gap: var(--spacing-3, 12px);
+  gap: var(--spacing-3, 12px);
```
[the remark this answers is not in the document]
Comment (KentMossback) [reply libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss]: Removed and changed to gap: fixed.$spacing-3;

File: libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss [hunk 4]
```diff
@@ -0,0 +1,85 @@
+@use '../../../../style-dictionary/dist/scss/fixed-variables.scss' as fixed;
+@use '../../text-styles.scss';
+
+.container {
+  display: flex;
+  width: 340px;
```
[the remark this answers is not in the document]
Comment (KentMossback) [reply libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss]: Set width to 100% to adjust to parent container

File: libs/pxweb2-ui/src/lib/components/Select/Select.tsx (hunk 1 above)
Comment (KentMossback) [reply libs/pxweb2-ui/src/lib/components/Select/Select.tsx:238]: Removed commented code

File: libs/pxweb2-ui/src/lib/components/Select/Select.tsx (hunk 2 above)
Comment (KentMossback) [reply libs/pxweb2-ui/src/lib/components/Select/Select.tsx:228]: Formatted code as suggested

**Comment by KentMossback** [comment]
> When I look in Storybook it looks good, but when I go to the branch preview and look at it in the variablebox the checked mark is not centered in the middle of the circle.
> 

As Sjur mentioned did the button not look good at all inside the variablebox. Now the style is set on the input radio element.

File: libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  padding: 10px var(--spacing-2, 8px);
```
[the remark this answers is not in the document]
Comment (KentMossback) [reply libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss]: Did as you suggested

File: libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  padding: 10px fixed.$spacing-2;
+  position: relative;
+  padding-left: 44px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-family: PxWeb-font-400;
+  letter-spacing: 0em;
+  color: var(--px-color-text-default);
+  line-height: 24px;
+  font-size: 16px;
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss:25]: When we set values for CSS for fonts, we should use REM and not PX. Both line 24 and 25 here.

With REM the users can decide what font sizes they want in the settings of the browser, so if they set "Large" font-size, all sizes increase on any website. PX does not adjust itself in that way.

**Review by SjurSutterudSagen — CHANGES_REQUESTED** [review]

File: libs/pxweb2-ui/src/lib/components/Select/Select.tsx [hunk 7]
```diff
@@ -219,26 +224,17 @@ function VariableBoxSelect({
...
-                  onChange={() => {
-                    setClickedItem(option);
-                  }}
-                />
-                <Label htmlFor={option.value}>{option.label}</Label>
-              </div>
-            ))}
-          </div>
-        </Modal>
+        >
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Select/Select.tsx:227]: The indentation is off

**Review by SjurSutterudSagen — CHANGES_REQUESTED** [review]

**Review by SjurSutterudSagen — APPROVED** [review]
Looks good now, good job :)

**Review by MikaelNordberg — APPROVED** [review]
Looks good

**Pull request merged**